collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -439,6 +481,16 @@ def data(self):
             'power_draw': self.power_draw(),
         }
 
+        pci_bw_max = self.pci_bw_max()
+        if not pci_bw_max:
+            ...
```

Comment:
```suggestion
        if not pci_bw_max:
```

`...` was just a placeholder for code, we don't need it.